File: Python_Virtual_World/Notifications.py
class Notifications:

    def __init__(self):

        self.__notifications = []


    def add(self, notification : str):

        self.__notifications.append(notification)

    def clear(self):

        self.__notifications.clear()

    def load(self) -> str:

        out = ""
        for node in self.__notifications:

            out += f"{node}\n"

        return out

File: Java_Virtual_World/src/Gameplay/Notifications.java
package Gameplay;
import java.util.Vector;
public class Notifications {
    final private Vector<String> nots;
    public Notifications(){
        nots = new Vector<String>();
    }
    public void addNode(String node){
        nots.add(node);
    }
    public void clearNots(){
        nots.clear();
    }
    public String loadNots(){
        StringBuilder out = new StringBuilder();
        for(String node : nots){
            out.append(node).append("\n");
        }
        return out.toString();
    }
}
